releasenotes/notes/feat-llmobs-asyncio-e8247cfdc720a1a9.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    LLM Observability: Introduces automatic tracing context propagation for LLM Observability traces involving asynchronous tasks created via ``asyncio.create_task()``.
```

Comment:
might be worth adding it works for `futures` too? (tbh i'm not too well-versed on how they relate haha)

Reply:
We already have this for futures, this PR just enables it automatically on `LLMObs.enable()`, but agreed will add a separate note!